spyder/widgets/tabs.py

```diff
@@ -256,16 +257,28 @@ def __init__(self, parent, actions=None, menu=None,
         self.corner_widgets = {}
         self.menu_use_tooltips = menu_use_tooltips
 
-        self.setStyleSheet("QTabWidget::tab-bar {"
-                           "alignment: left;}")
-
         if menu is None:
             self.menu = QMenu(self)
             if actions:
                 add_actions(self.menu, actions)
         else:
             self.menu = menu
 
+        # QTabBar forces the corner widgets to be smaller than they should on
+        # some plugins, like History. The top margin added allows the
+        # toolbuttons to expand to their normal size.
+        # See: spyder-ide/spyder#13600
+        top_margin = 8 if os.name == "nt" else 5

```

Comment:
Checking again seems like in a resolution of 1366 x 768 a value of 9 looks better, but testing in a screen with 1920 x 1080 seems like 5 is better (if 9 is set, the misalignment appears again with the higher resolution):

1366 x 768
![tlow](https://user-images.githubusercontent.com/16781833/91762935-20bdf680-eb9b-11ea-9945-00dfb8ad9b79.gif)


1920 x 1080
![thigh](https://user-images.githubusercontent.com/16781833/91762866-1c91d900-eb9b-11ea-9423-973cd9374913.gif)


What do you think we can do to prevent this? Maybe qss style sheets allow media queries?

Reply:
>Maybe qss style sheets allow media queries?

No, QSS does not include media queries as far as I can tell. (https://doc.qt.io/Qt-5/stylesheet-syntax.html)

The hamburger button being misaligned is an issue since 1.x, I think the problem at hand was actually having the tool button be smaller.

I tried to improve the misalignment as it is something that I think would make the UI consistent, but that fix should be addressed separately.

Reply:
>Ok 👍 I thought that one of the goals of the new API was to solve such UI inconsistency

It does not fix it, but it will make fixing it much easier, since now the API is in charge of the hamburger button and providing a unified way of adding buttons and widgets to the toolbar of a panel. We can tweak a value in a single place, instead of having to tweak every single plugin to try to "manually (and unsuccessfully) recreate" the layout, as is the case of 4.x and before. 

>if you have any idea of how to prevent such an issue let us know

Besides manual tweaking and having it look best at a given resolution, I do not see any other way around it as the QSS to style the TabBar of a TabWidget is not as flexible as other widgets.

Reply:
>Also please change this to be 9 instead of 8 on Windows then

Fixed.